src/NumericalAlgorithms/LinearSolver/ExplicitInverse.hpp

```diff
@@ -54,7 +54,6 @@ struct ExplicitInverse {
  * cost, but all successive solves converge immediately.
  *
  * \par Advice on using this linear solver:
- *
```

Comment:
this change should be in the previous commit?

Reply:
Fixed